.buildkite/dagster-buildkite/dagster_buildkite/steps/test_project.py

```diff
@@ -8,15 +8,17 @@
 from ..step_builder import CommandStepBuilder
 from ..utils import BuildkiteLeafStep, GroupStep
 
+build_for: Set[AvailablePythonVersion] = set()
+build_core_for: Set[AvailablePythonVersion] = set()
```

Comment:
comment re-enforcing assumptions & more verbose name given the level of shenanigans at play